backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -9,10 +9,11 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. OADP is also capable of backing up both containerized applications and virtual machines (VMs).
+The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering {product-title} applications, application-related cluster resources, persistent volumes, and internal images. {oadp-short} is also capable of backing up both containerized applications and virtual machines (VMs).
 
-However, OADP does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or OpenShift Operators.
+However, {oadp-short} does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or {OCP-short} Operators.
 
+{oadp-short} support is provided to customer workload namespaces only. Full cluster backup and restore are not supported.
```

Comment:
maybe support is provided to customer workload namespaces, and cluster scope resources. support is provided to customer workload namespaces 
maybe link to the cluster backup and restore from here.

Reply:
@amastbau - please review again, most wise and powerful one!